src/components/Table/Table.jsx

```diff
@@ -511,41 +479,194 @@ const Table = createClass({
 		className: string,
 
 		/**
-		 * Adjusts the style of the table to have more spacing within the table cells
+		 * Adjusts the row density of the table to have more or less spacing.
+		 */
+		density: oneOf(['compressed', 'extended']),
```

Comment:
The only downside to doing that is the docs don't properly show the string values.